src/api/app/views/layouts/webui/_footer.html.haml

```diff
@@ -21,7 +21,7 @@
     %ul
       %li= link_to('OpenBuildService.org', 'https://openbuildservice.org/')
       %li= link_to('Documentation', 'https://openbuildservice.org/help/manuals/')
-      %li= link_to('API Documentation', apidocs_index_path)
+      %li= link_to('API Documentation', '/apidocs/')
```

Comment:
This will break if OBS is served with some prefix like we do in the review apps with `RAILS_RELATIVE_URL_ROOT` for instance.

Reply:
I replaced it with the absolute path. Is that better?

Reply:
@hellcp-work  please have another look.